test/kubetest2-capi/README.md

```diff
@@ -0,0 +1,12 @@
+# Kubetest2 Cluster API Deployer
+
+This component of kubetest2 is responsible for test cluster lifecycles for clusters deployed with Cluster API.
+
+## Usage
+
+As an example, to deploy with CAPD:
+
+```
+export KIND_EXPERIMENTAL_DOCKER_NETWORK=bridge
+kubetest2 capi --provider docker --repo-root $CLONEDREPOPATH --build --up --down
```

Comment:
just curious, have you tested this with any other providers?

Reply:
The `--build` parts won't work with other providers today, it only (barely) works with kind and CAPD because CAPD relies on kind images. The `Build` method on the provider just delegates to the Kind provider, if we really wanted it to work seamlessly we'd probably need to go all the way to integrating image-builder.

Reply:
Oh and specifically to answer your question, I tested just the `--up` parts with AWS as well :). This deployer mostly just delegates to `kind` and `clusterctl`, so it should work with the same caveats those tools have, ie they need to be on the path, there's some magic environment variables, etc

Reply:
After this merges in, I'm intending to open a PR to the AWS Cloud Provider repo to start testing the out of tree CPI.